src/raster/r.survey/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is not needed at r.survey now stays in the main GRASS GIS Addon repo.

Reply:
Ok.. thank you
Done